### DISCUSSION TITLE
ERROR on proc 0: DeePMD-kit C API Error: DeePMD-kit Error: TensorFlow Error: NOT_FOUND: graph-compress.pb; No such file or directory

### DISCUSSION THREAD

**Discussion by chb-deep** [deepmodeling/deepmd-kit#2703] [Q&A] [answered]
conda create deepmd-kit=*=*cpu libdeepmd=*=*cpu lammps -c https://conda.deepmodeling.com -n deepmd -y
I installed deepmd-kit and lammps with the above commands,but  I get an error when using lmp
Before running the task, the deepmd environment has been loaded. "conda activate deepmd"
How can I solve this problem? Can anyone help me?

[shuoxing@node01 02.lmp]$ conda activate deepmd
(deepmd) [shuoxing@node01 02.lmp]$ lmp -in in.lammps 
LAMMPS (23 Jun 2022 - Update 4)
OMP_NUM_THREADS environment is not set. Defaulting to 1 thread. (src/comm.cpp:98)
  using 1 OpenMP thread(s) per MPI task
Loaded 1 plugins from /home/shuoxing/.conda/envs/deepmd/lib/deepmd_lmp
Reading data file ...
  triclinic box = (0 0 0) to (10.114259 10.263124 10.216793) with tilt (0.036749877 0.13833062 -0.056322169)
  1 by 1 by 1 MPI processor grid
  reading atoms ...
  5 atoms
  read_data CPU = 0.000 seconds
DeePMD-kit WARNING: Environmental variable TF_INTRA_OP_PARALLELISM_THREADS is not set. Tune TF_INTRA_OP_PARALLELISM_THREADS for the best performance. See https://deepmd.rtfd.io/parallelism/ for more information.
DeePMD-kit WARNING: Environmental variable TF_INTER_OP_PARALLELISM_THREADS is not set. Tune TF_INTER_OP_PARALLELISM_THREADS for the best performance. See https://deepmd.rtfd.io/parallelism/ for more information.
DeePMD-kit WARNING: Environmental variable OMP_NUM_THREADS is not set. Tune OMP_NUM_THREADS for the best performance. See https://deepmd.rtfd.io/parallelism/ for more information.
Summary of lammps deepmd module ...
  >>> Info of deepmd-kit:
  installed to:       /home/shuoxing/.conda/envs/deepmd
  source:             v2.2.2
  source branch:       HEAD
  source commit:      92ca097b
  source commit at:   2023-05-24 13:45:03 +0800
  surpport model ver.:1.1 
  build variant:      cpu
  build with tf inc:  /home/shuoxing/.conda/envs/deepmd/include;/home/shuoxing/.conda/envs/deepmd/include
  build with tf lib:  /home/shuoxing/.conda/envs/deepmd/lib/libtensorflow_cc.so
  set tf intra_op_parallelism_threads: 0
  set tf inter_op_parallelism_threads: 0
  >>> Info of lammps module:
  use deepmd-kit at:  /home/shuoxing/.conda/envs/deepmdNOT_FOUND: graph-compress.pb; No such file or directory
ERROR on proc 0: DeePMD-kit C API Error: DeePMD-kit Error: TensorFlow Error: NOT_FOUND: graph-compress.pb; No such file or directory (/home/conda/feedstock_root/build_artifacts/libdeepmd_1689710008919/work/source/lmp/pair_deepmd.cpp:838)
Last command: pair_style	deepmd graph-compress.pb
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 0
[unset]: write_line error; fd=-1 buf=:cmd=abort exitcode=1
:
system msg for write_line failure : Bad file descriptor

in.lammps:
# gas phase methane

units           metal
boundary        p p p
atom_style      atomic

neighbor        1.0 bin
neigh_modify    every 10 delay 0 check no

read_data	conf.lmp
mass 		1 1
mass		2 12

pair_style	deepmd graph-compress.pb
pair_coeff	* *

velocity        all create 50.0 23456789
fix             1 all nvt temp 50.0 50.0 0.5
timestep        0.001

thermo_style    custom step pe ke etotal temp press vol
thermo          100
dump		1 all custom 100 ch4.dump id type x y z 

run             5000

log.lammps:
LAMMPS (23 Jun 2022 - Update 4)
OMP_NUM_THREADS environment is not set. Defaulting to 1 thread. (src/comm.cpp:98)
  using 1 OpenMP thread(s) per MPI task
Loaded 1 plugins from /home/shuoxing/.conda/envs/deepmd/lib/deepmd_lmp
# gas phase methane

units           metal
boundary        p p p
atom_style      atomic

neighbor        1.0 bin
neigh_modify    every 10 delay 0 check no

read_data	conf.lmp
Reading data file ...
  triclinic box = (0 0 0) to (10.114259 10.263124 10.216793) with tilt (0.036749877 0.13833062 -0.056322169)
  1 by 1 by 1 MPI processor grid
  reading atoms ...
  5 atoms
  read_data CPU = 0.000 seconds
mass 		1 1
mass		2 12

pair_style	deepmd graph-compress.pb
Summary of lammps deepmd module ...
  >>> Info of deepmd-kit:
  installed to:       /home/shuoxing/.conda/envs/deepmd
  source:             v2.2.2
  source branch:       HEAD
  source commit:      92ca097b
  source commit at:   2023-05-24 13:45:03 +0800
  surpport model ver.:1.1 
  build variant:      cpu
  build with tf inc:  /home/shuoxing/.conda/envs/deepmd/include;/home/shuoxing/.conda/envs/deepmd/include
  build with tf lib:  /home/shuoxing/.conda/envs/deepmd/lib/libtensorflow_cc.so
  set tf intra_op_parallelism_threads: 0
  set tf inter_op_parallelism_threads: 0
  >>> Info of lammps module:
  use deepmd-kit at:  /home/shuoxing/.conda/envs/deepmdERROR on proc 0: DeePMD-kit C API Error: DeePMD-kit Error: TensorFlow Error: NOT_FOUND: graph-compress.pb; No such file or directory (/home/conda/feedstock_root/build_artifacts/libdeepmd_1689710008919/work/source/lmp/pair_deepmd.cpp:838)
Last command: pair_style	deepmd graph-compress.pb

**Comment by njzjz** [comment, marked as answer]
I think the error message is clear enough.

**Reply by chb-deep** [reply]
I have fixed this issue, thank you for the answer

**Reply by hezs-LZJTU** [reply]
Hi, I also encountered this problem. How did you solve it?
